antora/README.md

```diff
@@ -1,29 +1,35 @@
 # Enterprise Contract website
 
-This directory contains the configuration needed by [Antora][antora] to render
-the content of the `docs` directory and documentation sources from other
-directories as HTML.
+These are the sources for building the https://enterprisecontract.dev website.
+The website uses [Hugo](https://gohugo.io/) and
+[Antora][https://docs.antora.org/] to build different sections of the website.
 
-Run `npm run build` to generate the rendered HTML files in the `public`
-directory.
+## Live reload preview
 
-## Local development
+To run the preview of the website as changes are made to files within this
+repository run `make preview`. Note that the `preview` target also invokes
+`npm ci` in the `antora` directory which will reinstall and remove linked
+packages needed for development, running `npm run dev:setup` should restore the
+links and help with debugging. More on this below.
 
-Antora supports building from local git clone of the repository and with
-uncommited changes. This is done by changing the `content/sources` in
-`antora-playbook.yml` to point to the local directory for the `url` and `HEAD`
-for `branches.
+## Development
 
-With `hack/local-build.sh` script performs this by creating a temporary playbook
-file pointing to the local clones of the mentioned repositories if those
-repositories are located in the `../<repository>` for a
-`https://github.com/<org>/<repository>.git` git URL. For example, it is expected
-that the `ec-policies` repository is located at `../ec-policies`. If the local
-repository is on a different path create a symlink pointing from
-`../<repository>` to that path.
+The website makes makes use of
+[Antora Extensions](https://docs.antora.org/antora/latest/extend/extensions/),
+there are several extensions defined as development dependencies in
+`antora/package.json` and configured in `antora/antora-playbook.yml`.
 
-The `hack/local-live.sh` script calls `hack/local-build.sh` for any changes to
-files in the locally cloned repositories. Useful to leave in the background and
-have the generated HTML site continuously updated.
+To make changes locally to the extensions run `npm run dev:setup` script in the
```

Comment:
I'm probably not doing this right. If I introduce  a typo in `rego-antora-extension/index.js` on the linked ec-cli repo, the build is still successful.